src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Microsoft.Azure.Commands.ResourceManager.Cmdlets.dll-Help.xml

```diff
@@ -1115,6 +1137,37 @@
           </command:commandLine>
         </command:commandLines>
       </command:example>
+    <command:example>
+        <maml:title>-------  Example 2 -------</maml:title>
```

Comment:
nit: >--------------------------  Example 2 --------------------------</ to meet with other cases above